cmd/podman/varlink/io.podman.varlink

```diff
@@ -3,6 +3,33 @@
 interface io.podman
 
 
+# Version is the structure returned by GetVersion
```

Comment:
Is this a mistake? Didn't this disappear in the earlier varlink PR?

Reply:
fixed, likely due to rebases